pkgs/applications/office/gnucash/default.nix

```diff
@@ -119,11 +92,10 @@ stdenv.mkDerivation rec {
   # and error based on failing tests.
   doCheck = true;
   preCheck = ''
-    make \
+    make --jobs=$NIX_BUILD_CORES \
```

Comment:
This should be filtered according to the value of `enableParallelBuilding`; something like `lib.optionalString`

Reply:
Good idea, I looked into this and

- It should be `enableParallelChecking` instead, right?
- `enableParallelChecking` seems to default to false.
- According to [upstream docs](https://wiki.gnucash.org/wiki/Testing), we should build `check` instead of `test`, and we should be able to remove the entire `preCheck` list.
- Not sure why it's selecting `test` over `check`, [`setup.sh`](https://github.com/NixOS/nixpkgs/blob/d6b6b5f94f4dfe2e24583bdc2235b493afdc0095/pkgs/stdenv/generic/setup.sh#L1325-L1328) seems to prioritize `check` over `test`.

Anyway, these are the changes before squashing: https://github.com/nevivurn/nixpkgs/commit/8d46780c6b59c7009aa0506fcf1a9f26a2099180